<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shaderRenamerGUI</class>
 <widget class="QMainWindow" name="shaderRenamerGUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="mid_HBoxLayout">
      <item>
       <layout class="QGridLayout" name="midLeft_GridLayout">
        <item row="0" column="0">
         <widget class="QListView" name="shadersListView">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="bottom_VBoxLayout">
      <item>
       <widget class="QLabel" name="assignedGeoLable">
        <property name="text">
         <string>geo assigned:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="geoAssignedListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="exportButton_HBoxLayout">
        <item>
         <widget class="QPushButton" name="cancel_Button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="renameButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color: rgb(150, 200, 150);}</string>
          </property>
          <property name="text">
           <string>Rename Shaders</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="top_GridLayout" columnstretch="0,0">
      <item row="1" column="0">
       <widget class="QLineEdit" name="assetLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="variationLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="shadersLable">
        <property name="text">
         <string>shaders:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="assetLabel">
        <property name="text">
         <string>Asset:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="variationLable">
        <property name="text">
         <string>Variation:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
